lib/chemistrykit/reporting/html_report_assembler.rb

```diff
@@ -27,12 +48,31 @@ def assemble
             @total_reactions += element['data-count'].to_i
             @total_failures += element['data-failures'].to_i
             @total_pending += element['data-pendings'].to_i
-            @total_duration += element['data-duration'].to_f
           end
 
           doc.css('.example-group').each do |group|
-            @groups << group
+            beaker = group.css('h3')
+            beaker_name = beaker.text.split(/\n/).last
+            unless @testsuites.has_key?(beaker_name)
+              @testsuites[beaker_name] = ['passing', []]
+            end
+            group.css('.example').each do |example|
+              @testsuites[beaker_name][1] << example
+              if example.attr('class').to_s.include? 'failing'
+                @testsuites[beaker_name][0] = 'failing'
+              end
```

Comment:
if a particular test is pending, I will mark the beaker as "passing" because technically nothing failed. It will still show up as pending in the html report and junit results. The beaker box will just be green.